pkg/kotsadmsnapshot/backup.go

```diff
@@ -930,3 +949,79 @@ func prepareIncludedNamespaces(namespaces []string) []string {
 	}
 	return includedNamespaces
 }
+
+// excludeShutdownPodsFromBackup will exclude pods that are in a shutdown state from the backup
+// this is to prevent the hook backup from failing if a pod is in a shutdown state and cannot be backed up
```

Comment:
this second statement is not necessary since i believe there are other scenarios that would cause the backup to fail